### PR TITLE
Add PagerDuty to AlertManager channels

### PR DESCRIPTION
**Pull request by jb3**
I wanted to be alerted via mobile when AlertManager things trip, so I've added a PagerDuty configuration.

### CODE REVIEW

File: kubernetes/namespaces/monitoring/alerts/alertmanager.yaml [hunk 1]
```diff
@@ -8,17 +8,20 @@ route:
...
-      {{ end }}{{ else }}Alert has resolved.{{ end }}
-    fields:
-    - title: Alert
-      value: "{{ .GroupLabels.alertname }}"
+          {{ end }}{{ else }}Alert has resolved.{{ end }}
+        fields:
+          - title: Alert
+            value: "{{ .GroupLabels.alertname }}"
+    pagerduty_configs:
+      - routing_key_file: "/opt/pydis/alertmanager/webhooks/PAGERDUTY_KEY"
```
Comment (jchristgit) [inline kubernetes/namespaces/monitoring/alerts/alertmanager.yaml:26]: How does the PagerDuty key get there? Shouldn't we use `git-crypt` for this?

**Review by jchristgit — CHANGES_REQUESTED** [review]
I think the secret should live in the repo using our tried and trusted method.

**Review by jchristgit — APPROVED** [review]
Please excuse my earlier review. My dog walked over my keyboard. Of course, this looks perfectly fine, and external influences forcing me to approve this pull request are completely absent.

**Comment by jb3** [comment]
i click the wrong button and create merge commit whoopsie i will fix history later to remove that

**Pull request merged**